src/interfaces/ViewInterface/ViewInterfaceContainer.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { get, noop } from 'lodash';
+import { Icon } from '@folio/stripes/components';
+import ViewInterface from './ViewInterface';
+import { interfaceResource } from '../../common/resources';
+
+export class ViewInterfaceContainer extends Component { // eslint-disable-line import/prefer-default-export
```

Comment:
I don't see the reason to use here `eslint-disable-line import/prefer-default-export`